common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -32,6 +32,15 @@
 from embargo.test_utils import restrict_course
 
 
+class UrlMixin(object):
+    """
+    Mixins for URL processing
+    """
+    def get_absolute_url(self, path):
+        """ Generate an absolute URL for a resource on the test server. """
+        return u'http://testserver/{}'.format(path.lstrip('/'))
+
+
```

Comment:
@jimabramson I understand. I think the primary purpose for the mixin was the lack of a request object to call build_absolute_uri on... what would be your recommendation in that regard?